tests/typowalking/wordwalker.js

```diff
@@ -293,7 +293,7 @@ bender.test( {
 			wordsReturned;
 
 		bot.setHtmlWithSelection(
-			'<p>paragraph<br/>break<sup>superscript</sup> paragraph<sub>subscript</sub>'
+			'<p>paragraph<br/>break<sup>superscript</sup> paragraph<sub>subscript</sub></p>'
```

Comment:
pretty sure it was unintentional given the test title.  it does fix itself though